cms/static/js/views/license-selector.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Provide license selector view for license models.
+ * These license selectors are used to set the license for
+ * new or existing courses and videos.
+ */
+define(["js/views/baseview", "underscore", "gettext", "js/models/license", "js/views/license-view"],
+    function(BaseView, _, gettext, LicenseModel, LicenseView) {
+
+        var LicenseSelector = BaseView.extend({
+            events: {
+                "click .license-button" : "onLicenseButtonClick",
+            },
+
+            initialize: function(options) {
+                this.template = this.loadTemplate("license-selector");
+                if (!this.model) {
+                    this.model = new LicenseModel();
```

Comment:
I don't think it's worth it to have a default model like this. If you instantiate this view without passing a model to it, something is *wrong*, and that should be detected as early as possible. If you really want to support that case, then you better have several unit tests that cover that case.